ruby.rb

```diff
@@ -3,49 +3,51 @@ class ArticlesController < ApplicationController
 
   def index
```

Comment:
You should change the name of this class to `app/controllers/articles_controller.rb`